src/sentry/auth/helper.py

```diff
@@ -121,6 +119,8 @@ def pipeline_is_valid(self):
         session = self.request.session.get('auth', {})
         if not session:
             return False
+        if session.get('flow') not in (self.FLOW_LOGIN, self.FLOW_SETUP_PROVIDER):
```

Comment:
this will force the no longer supported flow to re-set and error out on the user, which is a slight inconvenience but not a big deal